src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/sendtofirsttier/SendToFirstTierSubmittedHandlerTest.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.sendtofirsttier;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType.MID_EVENT;
+import static uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType.SUBMITTED;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.service.CcdCallbackMapService;
+
+@ExtendWith(MockitoExtension.class)
+public class SendToFirstTierSubmittedHandlerTest {
+
+    private static final String USER_AUTHORISATION = "Bearer token";
+    public static final long CASE_ID = 1234L;
+
+    private SendToFirstTierSubmittedHandler handler;
+
+    @Mock
+    private CcdCallbackMapService ccdCallbackMapService;
+
+    @Mock
+    private Callback<SscsCaseData> callback;
+
+    @Mock
+    private CaseDetails<SscsCaseData> caseDetails;
+
+
+    private SscsCaseData caseData;
+
+    @BeforeEach
+    void setUp() {
+        handler = new SendToFirstTierSubmittedHandler(ccdCallbackMapService, true, true);
+
+        caseData = SscsCaseData.builder()
+                .ccdCaseId(String.valueOf(CASE_ID))
+                .postHearing(PostHearing.builder()
+                        .sendToFirstTier(SendToFirstTier.builder().build())
+                        .build())
```

Comment:
check the tabbing in this file, seems like its twice what its meant to be

Reply:
Done